### PR TITLE
SEQNG-234: AmpCount should be written as a string

### PR DESCRIPTION
**Pull request by cquiroz**
The AmpCount values should be written as a String according to the seqexec LUT

### CODE REVIEW

File: modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala [hunk 1]
```diff
@@ -44,10 +44,10 @@ object GmosControllerEpics extends GmosSouthController {
...
   implicit val ampGainSettingEncoder: EncodeEpicsValue[AmpGainSetting, String] = EncodeEpicsValue(_.value.toString)
 
-  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, Int] = EncodeEpicsValue {
-    case AmpCount.THREE  => 3
-    case AmpCount.SIX    => 6
-    case AmpCount.TWELVE => 12
+  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, String] = EncodeEpicsValue {
+    case AmpCount.THREE  => "BEST"
+    case AmpCount.SIX    => "ALL"
+    case AmpCount.TWELVE => "SECONDARY"
```
Comment (jluhrs) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala]: Contents of gmosAmpCount.lut:
#OT amp count	GMOS amp mode
Six				BEST
Twelve			ALL

Comment (jluhrs) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala]: THREE is not an option in OT.

Comment (cquiroz) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala]: hmm, I missunderstood. THREE is in the OCS code, maybe hidden in the UI?

Comment (sraaphorst) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala]: Yes, I just worked on this, and THREE is an option in the OCS code, but only for, IIRC, E2V detectors. I think now that we are using Hamamatsu CCDs for both GMOS-N and GMOS-S, THREE may be obsolete, but I am not certain.

**Pull request merged**